packages/mobile/src/screens/track-screen/TrackScreenDetailsTile.tsx

```diff
@@ -312,17 +311,14 @@ export const TrackScreenDetailsTile = ({
       }
       hideFavorite={hideFavorite}
       hideRepost={hideRepost}
-      hideShare={(is_unlisted && !isOwner) || !field_visibility?.share}
```

Comment:
Really a product question, but do we want to stop respecting existing field visibility settings, or just take away the ability to set new ones?

I def understand doing it for collections and new track uploads, but I could see someone getting annoyed if they specifically set it not to share before.

Probably small potatoes and nobody cares, but just checking to make sure.

Reply:
i think on the call yesterday we decided to leave it in the backend, but remove it from the front-end